src/hyp3_back_projection/dem.py

```diff
@@ -0,0 +1,52 @@
+"""Prepare a Copernicus GLO-30 DEM virtual raster (VRT) covering a given geometry"""
+import logging
+from pathlib import Path
+
+import requests
+from shapely.geometry import Polygon
+
+from hyp3_back_projection import utils
+
+
+log = logging.getLogger(__name__)
+
+
+def ensure_egm_model_available():
+    """Ensure the EGM module is available.
+    Currently using the EGM2004 model provided by Stanford, but hope to switch to a public source.
+    """
+    url = 'https://ffwilliams2-shenanigans.s3.us-west-2.amazonaws.com/lavas/egm2008_geoid_grid'
```

Comment:
would this be better as a global variable? It seems we generally style these types of paths as global variables, but it is also fine hard-coded here.